### PR TITLE
Added Jupyter notebooks

### PR DESCRIPTION
**Pull request by mdwyman**
Summary of changes:
[use_bluesky.sh](https://github.com/BCDA-APS/use_bluesky/compare/master...jupyter#diff-9749da12fa71fedd551b20cf22966eee) -- added argument "-jupyter" otherwise runs as it did previously
[install_jupyter_kernel](https://github.com/BCDA-APS/use_bluesky/commit/cf039a41179656c1658adf69e7a3b19cd8531b81) -- creates kernel.json so that notebooks will use a bluesky-style ipython profile
[install_bluesky_profile_for_ipython.sh](https://github.com/BCDA-APS/use_bluesky/compare/master...jupyter#diff-44fb070a7d9df2c32ca3d672dd1bb052) -- creates a second profile for use in jupyter notebooks
[00-startup-jup.py](https://github.com/BCDA-APS/use_bluesky/compare/master...jupyter#diff-4728d0ab495f46b04bbc9b4b2d4af7c0) -- imports proper event handler (install_nb_kicker) for jupyter notebooks

### CODE REVIEW

**Comment by mdwyman** [comment]
@prjemian 

Separated kernel creation into new script; updated "Summary of Changes" in original post

File: starters/use_bluesky.sh [hunk 1]
```diff
@@ -38,10 +45,26 @@ export BLUESKY_PROFILE=${1:-$BLUESKY_PROFILE__INTERNAL_DEFAULT}
...
+		general_startup
+		echo "Starting jupyter-notebook"
+		$BLUESKY_BIN_DIR/jupyter-notebook
+		read -p "End of BlueSky console session - press return"
+	;;
+	*)
+		echo "Incorrect arguments. Either:"
+		echo "$ use_bluesky.sh "
+		echo "OR"
+		echo "$ uee_bluesky.sh -jupyter"
```
Comment (prjemian) [inline starters/use_bluesky.sh]: typo?

**Review by prjemian — COMMENTED** [review]
with that one change, this will be ready

**Comment by prjemian** [comment]
The changes with d483c10 were enough to merge this PR.
If you can remove the later changes (commits: bd7d807, 020ae53, ffe7e9b, e6aba84, 528220f) and commit them against #26, then a new PR will re-org the repository.

**Pull request merged**